app.js

```diff
@@ -41,49 +40,35 @@ app.configure(function() {
   });
 });
 
-app.post('/add-session', limiter, function (req, res) {
-  var formUrl = env.get("PROPOSAL_FORM_ACTION_URL");
-  var proposalUUID = uuid.v4();
-  var userInputs = {
-    firstName: req.body.firstName,
-    surname: req.body.surname,
-    email: req.body.email,
-    affiliationOrganization: req.body.affiliationOrganization,
-    otherFacilitators: req.body.otherFacilitators,
-    twitter: req.body.twitter,
-    space: req.body.space,
-
-    exhibitTitle: req.body.exhibitTitle,
-    exhibitMethod: req.body.exhibitMethod,
-    exhibitLink: req.body.exhibitLink,
-    exhibitDescription: req.body.exhibitDescription,
-    exhibitLearnReflect: req.body.exhibitLearnReflect,
-    exhibitWhyGoodMozfest: req.body.exhibitWhyGoodMozfest,
-    exhibitAnotherSpace: req.body.exhibitAnotherSpace,
+app.post(`/add-proposal`, limiter, function(req, res) {
+  // make sure keys are all lowercase and contain no symbols or spaces
+  // reference: https://www.npmjs.com/package/google-spreadsheet#row-based-api-limitations
+  var proposal = Object.assign({}, req.body, {
+    uuid: uuid.v4(),
+    githubissuenumber: ``,
+    timestamp: moment.utc().toString() // time in GMT (e.g., Fri May 26 2017 19:57:11 GMT+0000)
```

Comment:
code/docs inconsistency: utc is not gmt. utc has no concept of daylight saving time, whereas GMT does.